src/algebra/symmeterised.lean

```diff
@@ -0,0 +1,227 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.module.basic
+import tactic.abel
+
+/-!
+# Symmetrised algebra
```

Comment:
The filename has an extra e in. Also, given we have `localization`:

Reply:
Done.